cosmwasm/proto/axelarnet_add_chain.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package axelar.axelarnet.v1beta1;
+
+option go_package = "github.com/axelarnetwork/axelar-core/x/axelarnet/types";
+
+enum KeyType {
+  KEY_TYPE_UNSPECIFIED = 0;
+  KEY_TYPE_NONE = 1;
+  KEY_TYPE_THRESHOLD = 2;
+  KEY_TYPE_MULTISIG = 3;
+}
+
+message Chain {
+  string name = 1;
+  string native_asset_deprecated = 2;
+  bool supports_foreign_assets = 3;
+  KeyType key_type = 4;
+  string module = 5;
+}
+
+message AddCosmosBasedChainRequest {
```

Comment:
bad practice to duplicate, someone needs to maintain these proto definitions now. Either they should be fetched directly from the source dynamically, or just use JSON syntax implicitly like other proposal other types